frontend/public/components/catalog/catalog-items.jsx

```diff
@@ -211,6 +223,11 @@ export class CatalogTileViewPage extends React.Component {
         key={uid}
         onClick={() => this.openOverlay(item)}
         title={tileName}
+        badges={[
+          <CatalogTileBadge key={`${uid}${kind}`}>
```

Comment:
The `key` here could be a static string such as `type` since it's the only entry in the array.

Reply:
Done.